src/osmet/file.rs

```diff
@@ -210,3 +197,10 @@ fn verify_canonical(mappings: &[Mapping]) -> Result<u64> {
 
     Ok(cursor)
 }
+
+fn bincoder() -> impl bincode::Options {
+    // little endian is the default, but make it explicit
+    bincode::options()
+        .with_little_endian()
+        .allow_trailing_bytes()
```

Comment:
Reading https://github.com/servo/bincode/pull/323, we may also need to pin integer encoding, right?

Reply:
Sure, done.

Reply:
Oh sorry, I was actually asking whether we do rely on the old behavior for integer encoding, which doesn't seem to be the default anymore, or am I reading that wrong?

Reply:
We're accepting the breaking change to obtain possibly smaller file size.  osmet files are always generated by the exact build of coreos-installer that consumes them, so we don't have compatibility constraints.

Reply:
Ah, that's great!